mkdocs.yml

```diff
@@ -67,8 +67,6 @@ nav:
                 - Installation: distributions/gentoo/installation.md
           - Manjaro:
                 - Home: distributions/manjaro/home.md
-                - Building: distributions/manjaro/building.md
-                - FAQ: distributions/manjaro/faq.md
```

Comment:
I think removing this will create some dead links (i believe there's a link to it for updating kernel for wifi guide). It might also be good to have instructions to upgrade to a newer manjaro kernel for people who installed before your newer iso was out